sagemaker_neo_compilation_jobs/imageclassification_caltech/inference.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import json
+import mxnet as mx
+# Please make sure to import neomxnet
+import neomxnet  # noqa: F401
+import io
+import logging
+
+# Change the context to mx.gpu() if deploying to a GPU endpoint
+ctx = mx.cpu()
+
+def model_fn(model_dir):
+    logging.info('Invoking user-defined model_fn')
+    # The compiled model artifacts are saved with the prefix 'compiled'
+    sym, arg_params, aux_params = mx.model.load_checkpoint('compiled', 0)
```

Comment:
use `model_dir` in the prefix/path for `load_checkpoint`

Reply:
done